Cabal/Distribution/CabalSpecVersion.hs

```diff
@@ -30,20 +31,27 @@ cabalSpecFeatures CabalSpecV2_2  = Set.fromList
     [ Elif
     , CommonStanzas
     ]
+cabalSpecFeatures CabalSpecV2_4  = Set.fromList
+    [ Elif
+    , CommonStanzas
```

Comment:
Would it make sense to add `Globstar` here?

Reply:
Hmm. We could. I didn't think to because the parser is wholly agnostic to the pattern syntax. AFAICT, this 'feature' concept is only used to toggle bits of `D.PD.Parsec` on and off, depending on the version.

Reply:
Could you please implement this then if it's not too much trouble?